externs/olx.js

```diff
@@ -4310,6 +4311,14 @@ olx.layer.VectorOptions.prototype.source;
 
 
 /**
+ * Declutter images and text. Default is `false`.
```

Comment:
I am leaning towards `true` as default. Why would one want a cluttered map by default?

Reply:
Not sure. Note that decluttering not only affects labels, but also point symbols. What do others think?

Reply:
Ah OK, I didn't get that first. Considering that also point symbols are affected, I think `false` is a sane default.

Reply:
I'll try to change the default to `true` and check all the examples. If they all still look as expected (but uncluttered), then `true` it shall be 😄

Reply:
But then we also need a big note in the readme, don't you think? As I said, I changed my mind because of the point symbols that are affected… Would be nice to hear what other think.